### ISSUE TITLE
Adding a new task with just a link causes massive cpu usage and doesn't do anything

### ISSUE THREAD

**Issue by holeyness** [idpaterson/alfred-wunderlist-workflow#75]
The command I wrote in was: wl "url" e.g. wl www.google.com
By pressing enter (enter as a new task in inbox), it creates a python process of ~98% and doesn't actually add the url to my inbox list.
I have to kill it via killall.

**Comment by holeyness** [comment]
Nevermind, for some reason it fixed itself.

**Comment by idpaterson** [comment]
I think that was the sync task, the first run will be quite long and I've noticed very slow responses from the Wunderlist API today as well that could make it worse. After that initial sync the Wunderlist API nicely allows me to update only the branches that have been updated (i.e. if a task is added to one of your 10 lists, only that list gets re-synced rather than all your lists).

Did this noticeably lock up your computer or did you just happen to see that the process was running?
